GoogleForCreators/web-stories-wp
# Media: Fix video drag an drop upload in Safari

## Context



## Summary

This change fixes the issue with video upload from safari. Change logic in seekVideo, use seeking and seeked events. canPlay is not fired in safari. 

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. In safari. 
2. Open editor. 
3. Drag video into canvas. 
4. See it upload with preview. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10098
